Project.toml

```diff
@@ -8,12 +8,17 @@ SparseArrays = "2f01184e-e22b-5df5-ae63-d93ebab69eaf"
 SuiteSparse = "4607b0f0-06f3-5cda-b6b1-a6196a1729e9"
 
 [compat]
+BandedMatrices = "0.15, 0.17, 1"
```

Comment:
It's not needed, but it tests pass with 0.17. I included it just in case some package that wants to use `PDMats` but can't upgrade to `BandedMatrices` v1 due to conflicts.

Reply:
BandedMatrices is only a test dependency, and hence the compat entry does only apply when running the tests but is ignored when installing or depending on PDMats.

Reply:
Oh right, in that case this isn't necessary